packages/zoe/src/typeGuards.js

```diff
@@ -79,3 +121,231 @@ export const InvitationElementShape = M.splitRecord({
 export const OfferHandlerI = M.interface('OfferHandler', {
   handle: M.call(SeatShape).optional(M.any()).returns(M.string()),
 });
+
+export const SeatHandleAllocations = M.arrayOf(
+  harden({
+    seatHandle: SeatShape,
+    allocation: AmountKeywordRecordShape,
+  }),
+);
+
+export const ZoeMintI = M.interface('ZoeMint', {
+  getIssuerRecord: M.call().returns(IssuerRecordShape),
+  mintAndEscrow: M.call(AmountShape).returns(),
+  withdrawAndBurn: M.call(AmountShape).returns(),
+});
+
+export const ZcfMintI = M.interface('ZcfMint', {
+  getIssuerRecord: M.call().returns(IssuerRecordShape),
+  mintGains: M.call(AmountKeywordRecordShape, M.remotable('zcfSeat')).returns(),
+  burnLosses: M.call(
+    AmountKeywordRecordShape,
+    M.remotable('zcfSeat'),
+  ).returns(),
+});
+
+export const ExitObjectI = M.interface('Exit Object', {
+  exit: M.call().returns(),
+});
+
+export const InstanceAdminShape = M.remotable('InstanceAdmin');
+export const InstanceAdminI = M.interface('InstanceAdmin', {
+  makeInvitation: M.call(InvitationHandleShape, M.string())
+    .optional(M.any(), M.pattern())
```

Comment:
IIRC, the `customArgs` must be a CopyRecord
```suggestion
    .optional(M.record(), M.pattern())
```

Reply:
thanks.

Reply:
What about line 208?

Reply:
done now.